service/service.go

```diff
@@ -74,18 +74,9 @@ func GrpcConnection(client, server string, insecureAllowed bool) (*grpc.ClientCo
 	if isTLS {
 		// TODO: host may be insecure and will fail here; prompt for insecure or require flag
 
-		util.Logger.Debug("getting tls cert from cache...")
-		cert, err = getCertFromCache(grpcAddr)
-		if err != nil {
-			// TODO(waigani) check error
-			// return nil, errors.Trace(err)
+		// TODO(waigani) check error
+		// return nil, errors.Trace(err)
 
-			// if cert hasn't been cached, get a new one which caches it under the hood
-			util.Logger.Debug("no cert found, creating new one...")
-			if cert, err = newCert(grpcAddr); err != nil && !insecureAllowed {
-				return nil, errors.Trace(err)
-			}
-		}
 	}
 
 	conn, err := dial(grpcAddr, cert, insecureAllowed)
```

Comment:
no need to pass `cert` here – surely we can just just create the creds from `&tls.Config{}` where creds are needed?